async_simple/executors/SimpleExecutor.cpp

```diff
@@ -23,15 +23,11 @@ namespace executors {
 static constexpr int64_t kContextMask = 0x40000000;
 
 SimpleExecutor::SimpleExecutor(size_t threadNum) : _pool(threadNum) {
-    [[maybe_unused]] auto ret = _pool.start();
     assert(ret);
     _ioExecutor.init();
 }
 
-SimpleExecutor::~SimpleExecutor() {
-    _ioExecutor.destroy();
-    _pool.stop();
-}
+SimpleExecutor::~SimpleExecutor() { _ioExecutor.destroy(); }
```

Comment:
can guarantee all pushed task be executed before SimpleExecutor destroyed

Reply:
Yes, will drain off all the tasks when the pool destructed.